libs/apigateway-lambda-proxy/README.md

```diff
@@ -65,3 +65,85 @@ new ApiGatewayLambdaProxy(`my-api`, {
   },
 })
 ```
+
+### Monitoring
+
+```ts
+import {
+  ApiGatewayLambdaProxy,
+  RecommendedAlarms,
+} from '@wanews/pulumi-apigateway-lambda-proxy'
+
+const gw = new ApiGatewayLambdaProxy('apigw-prod', {
+  /* ... */
+})
+
+new RecommendedAlarms('alarms', {
+  snsTopicArn: 'arn:aws:sns:<region>:<account>:<topic>',
+  apiGateway: {
+    id: gw.apiGateway.id,
+    name: httpApi.name,
+    stage: gw.stage?.name,
+})
+```
+
+You can also create alarms for APIs created elsewhere:
+
+```ts
+import * as aws from '@pulumi/aws'
+import { RecommendedAlarms } from '@wanews/pulumi-apigateway-lambda-proxy'
+
+const httpApi = new aws.apigatewayv2.Api('apigw-http', {
+  /* ... */
+})
+
+new RecommendedAlarms('alarms', {
+  snsTopicArn: 'arn:aws:sns:<region>:<account>:<topic>',
+  apiGateway: {
+    id: httpApi.id,
+    name: httpApi.name,
+    stage: gw.stage?.name,
+  },
+})
+```
+
+Note that the name is only used for cosmetic purposes. You should set it to a value that easily identifies the API.
+
+By default, the following metrics are monitored:
+
+- 5xx error rate
+- 4xx error rate
+- integration latency
+
+However, if you're not happy with the defaults, you can override the default thresholds, or create individual alarms instead:
+
+```ts
+import {
+  ApiGatewayLambdaProxy,
+  RecommendedAlarms,
+} from '@wanews/pulumi-apigateway-lambda-proxy'
+
+const gw = new ApiGatewayLambdaProxy('apigw-prod', {
+  /* ... */
+})
+
+new RecommendedAlarms('alarms', {
+  snsTopicArn: 'arn:aws:sns:<region>:<account>:<topic>',
+  apiGateway: {
+    name: restApi.name,
+    stage: gw.stage?.name,
+  },
+  thresholds: {
+    errorRate5xxPercent: 0,
```

Comment:
Does this mean any 5xx will be alarmed or that the 500 percentage needs to hit 1% for an alarm to occur?

Reply:
Greater than or equal to zero. It sounds like you're implying that setting it to zero will result in an alarm that is always triggered?

And yes, I missed that, thanks! I'll update the docs to use a different example.

Reply:
Actually, it's probably a good idea to use `GreaterThanThreshold` instead of `GreaterThanOrEqualToThreshold `, so it still works as expected when the threshold 0 is provided.

Reply:
Did you make this change out of interest?